RecoEgamma/ElectronIdentification/test/runtests.sh

```diff
@@ -1,18 +1,18 @@
 function die { echo $1: status $2 ;  exit $2; }
 
 ids_to_test=(
-    
-    "RecoEgamma.ElectronIdentification.Identification.cutBasedElectronID_PHYS14_PU20bx25_V2_cff"
```

Comment:
cutBasedElectronID_PHYS14_PU20bx25_V2_cff is still in the release.
Why was it removed from the unit tests?
It's not an MVA ID that has a larger problem with removal.

same question about HEEPV60

Reply:
HEEP V60 added to unit tests. All PHYS14/CSA14 IDs are removed from release